OnnxBridge/LLAMA/sytorchBackendRep.py

```diff
@@ -137,45 +139,6 @@ def cleartext_post(code_list, program, scale, mode, indent):
     )
 
 
-def cleartext_fp_post(code_list, program, scale, mode, indent):
```

Comment:
changes related to cleartext_fp have been removed in your PR. Probably do a git pull from upstream to your fork.